packages/SidePanel/src/components/Footer/Footer.js

```diff
@@ -26,7 +29,14 @@ export default function Footer(props) {
   } = props;
 
   return (
-    <div data-pka-anchor="sidepanel.footer" css={footerCSS} {...{ isSticky, width }} {...moreProps}>
+    <div
+      data-pka-anchor="sidepanel.footer"
+      css={footerCSS}
```

Comment:
perhaps we should move away from using the `css` attribute approach by default and render the element as `<sc.sidePanelFooter>` instead and export that as `styled.div` in the css...

Reply:
👍 I'm using the new convention in the new added things. btw why it's `sc` not `styled` ?

Reply:
Imo `sc` (stands for styled component) makes it more readable rather than having `styled` written everywhere thats all.. open to discussion though. You can always update these existing elements with the approach too, doesn't have to just be new ones :)